projects/storefrontlib/src/lib/product/components/product-list/product-facet-navigation/product-facet-navigation.component.scss

```diff
@@ -33,11 +33,16 @@ $cx-search-facet-header-link-top: 10px !default;
 $cx-search-facet-header-link-icon-height: 2px !default;
 
 $cx-search-facet-header-icon-color: 'secondary' !default;
-$cx-search-facet-form-check-trbl-margin: 0 0 11px 0 !default;
 
 $cx-search-facet-label-color: 'secondary' !default;
 $cx-search-facet-label-selected-color: 'primary' !default;
-$cx-search-facet-label-top: 3px !default;
+
+$cx-search-facet-label-wrapper-trbl-padding: 0 0 0 0 !default;
```

Comment:
Is it not equivalent to the following `$cx-search-facet-label-wrapper-trbl-padding: 0 !default;`?

Reply:
of course it is, but as you can see, we use this `trbl` prefix (top, right, bottom, left), so I decided to go with this splitting zeros approach